plugins/octant/cmd/traceflow-plugin/main.go

```diff
@@ -0,0 +1,403 @@
+package main
```

Comment:
Question - is it possible to put Traceflow in the Antrea Octant main plugin, instead of creating a separate plugin? Like just add a new tab for Traceflow?

Reply:
> Question - is it possible to put Traceflow in the Antrea Octant main plugin, instead of creating a separate plugin? Like just add a new tab for Traceflow?

Yes, we can put Traceflow plugin and the previous plugin together.

Reply:
@jianjuns Do you prefer that we merge these two to one plugin?

Reply:
I feel a single plugin is easier for users to find out, with a single entry point for all Antrea views. But do you see any issue with that? We might want to seek other people's inputs as well.
@antoninbas @tnqn @McCodeman

Reply:
Agreed, I just didn't know it can be in one plugin.

Reply:
One plugin sounds good to me

Reply:
The advantage of separate octant-plugins is that if there is any issue with one plugin, the other will not be affected.
In this way, we can make sure that traceflow part will not affect previous monitoring CRD part.
I agree that one plugin is better for the long run, but consider the schedule and stability, I prefer to merge these two later.

Reply:
Sure we can merge later.

Reply:
Just to confirm, would it affect upgrade if from two plugins to one?